Jenkinsfile.sonar

```diff
@@ -20,7 +20,7 @@ pipeline {
 								sh """#!/bin/bash
 								${scannerHome}/bin/sonar-scanner -Dsonar.projectKey=LiskHQ_lisk-sdk \
 												 -Dsonar.organization=liskhq \
-												 -Dsonar.exclusions=**/test/** \
+												 -Dsonar.exclusions=**/test/**,**/migrations/**,reset_mem_tables.sql,**/protocol-specs/**,**/templates/browsertest.tmpl/** \
```

Comment:
Is this really specific to PRs? If not the same change needs to be made on line 35.

Reply:
We can safely ignore them by now in general should I added then on 35?

Reply:
I think it makes sense to have the same exclusion list for PRs (line 23) and branches (line 35)

Reply:
Added